srcpkgs/firefox/patches/fix-desktop-icon-path.patch

```diff
@@ -0,0 +1,11 @@
+--- taskcluster/docker/firefox-snap/firefox.desktop.orig	2019-01-18 19:31:39.428839442 +0100
++++ taskcluster/docker/firefox-snap/firefox.desktop	2019-01-18 19:32:20.689063456 +0100
+@@ -154,7 +154,7 @@
+ Terminal=false
+ X-MultipleArgs=false
+ Type=Application
+-Icon=/browser/chrome/icons/default/default128.png
++Icon=/usr/lib/firefox/browser/chrome/icons/default/default128.png
```

Comment:
Imo firefox would be more correct. It also would allow using the different resolution icons when sensible.

Reply:
thanks, seems to work